.buildkite/pipeline.package.mbp.yml

```diff
@@ -1,12 +1,46 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
 name: "fleet server package mbp"
-steps:
-  - label: ":pipeline: test step"
-    command: echo "test-pipeline"
+env:
+  REPO: 'fleet-server'
+  JOB_GCS_BUCKET: 'fleet-ci-artifacts'
+  SETUP_GVM_VERSION: 'v0.3.0'
+  DOCKER_REGISTRY: "docker.elastic.co"
 
-  - group: "test group"
+steps:
+  - group: "Package and Publish"
+    key: "package-publish"
     steps:
-      - label: "step1 of the group"
-        command: echo "step1 of the group"
+    - label: "Package Ubuntu-20 x86_64"
+      key: "package-x86-64"
+      command: ".buildkite/scripts/package.sh {{matrix.type}}"
+      agents:
+        provider: "gcp"
+        image: "family/core-ubuntu-2004"
+      matrix:
+        setup:
+          type:
+          - "snapshot"
+          - "staging"
+
+    - label: "Package Ubuntu-20 aarch64"
+      key: "package-arm"
+      command: ".buildkite/scripts/package.sh {{matrix.type}}"
+      agents:
+        provider: "aws"
+        image: "core-ubuntu-2004-aarch64"
+      matrix:
+        setup:
+          type:
+          - "snapshot"
+          - "staging"
 
-      - label: "step2 of the group"
-        command: echo "step2 of the group"
\ No newline at end of file
+  - label: "DRA snapshot"
+    key: "rda-snapshot"
+    command: ".buildkite/scripts/dra_snapshot.sh"
+    agents:
+      provider: "gcp"
+      image: "family/core-ubuntu-2004"
+    if: ${IS_FILE_EXISTS_IN_REPO}
```

Comment:
mmm when this step runs, post-checkout hook is not executed yet.
How does this variable get the value ?

Maybe those functions needs to be moved as part of the script `dra_snapshot.sh` ?

Reply:
If I'm not wrong, the post-checkout step is runningg before uploading the pipeline to the agent.
I tested it when I changed the file name value of this variable stayed `false`: https://github.com/elastic/fleet-server/pull/2702/commits/be236a42626a9b769847e55034ec7180dbd0e788